--- antlr3-3.5.2/tool/src/main/java/org/antlr/tool/ErrorManager.java.orig	2017-01-06 19:35:01.829424552 +0000
+++ antlr3-3.5.2/tool/src/main/java/org/antlr/tool/ErrorManager.java	2017-01-06 19:35:09.157151331 +0000
@@ -385,7 +385,6 @@
 			public void runTimeError(STMessage msg) {
 				switch (msg.error) {
 				case NO_SUCH_ATTRIBUTE:
-				case NO_SUCH_ATTRIBUTE_PASS_THROUGH:
 				case NO_SUCH_PROPERTY:
 					ErrorManager.warning(ErrorManager.MSG_STRING_TEMPLATE_ERROR, msg.toString());
 					return;
